BoolODE/simulator.py

```diff
@@ -140,5 +140,5 @@ def getInitialCondition(ss, ModelSpec, rnaIndex,
         pss = ((ModelSpec['pars']['r_' + genename])/\
                                       (ModelSpec['pars']['l_p_' + genename]))\
                                       *new_ics[revvarmapper['x_' + genename]]
-        new_ics[revvarmapper['p_' + genename.replace('_','')]] = pss
+        new_ics[revvarmapper['p_' + genename]] = pss
```

Comment:
I have viewed the version history for `simulator.py`. The line of code `new_ics[revvarmapper['p_' + genename.replace('_','')]] = pss` has not changed since the commit by Dr. Jalihal on March 29, 2020. In fact, the entire file has not experienced any changes since that commit. It is not clear why Dr. Jalihal made this choice. However, I have tracked the use of the method `getInitialCondition` within the 'BoolODE' package using PyCharm, and also looked at how these parameters are written to the `model.py` file. I see no evidence that maintaining the underscore in the gene name would hinder functionality. The only downside would be that when viewing the actual parameter names, the user may not find clear which underscores separate genes and which are just part of the gene name. However, it would be of no difference to the computer, which merely reads the value assigned to that parameter name.